bbz-project/src/components/ContextController.js

```diff
@@ -1,28 +1,39 @@
-import React, { createContext, useState } from "react";
+import React, { createContext, useState, useEffect } from "react";
 import useFetch from "../hooks/useFetch";
 
 const DataContext = createContext();
 
 const ContextController = (props) => {
     const [dbList, setDbList] = useState();
     const [dbNames, setDbNames] = useState();
-    const [currentDb, setCurrentDb] = useState({});
-    const { fetchDbNames, addRecord, deleteRecord, editRecord } = useFetch("https://backend-bbz.herokuapp.com/api")
+    const [currentDbId, setCurrentDbId] = useState(0);
+    const [currentDb, setCurrentDb] = useState([]);
+    const { get, addRecord, deleteRecord, editRecord } = useFetch("https://backend-bbz.herokuapp.com/api")
```

Comment:
If something is a placeholder for future features please provide a comment stating that near such code.

Reply:
Looks good to me.

Reply:
fixed